mne_bids_pipeline/config.py

```diff
@@ -216,7 +216,7 @@ class ArbitraryContrast(TypedDict):
     The ``emptyroom`` subject will be excluded automatically.
 """
 
-process_er: bool = False
+process_er: bool = True
```

Comment:
I think we should call this `process_empty_room` as `er` is pretty impossible to guess.